cmd/rootlesskit/main.go

```diff
@@ -313,14 +319,14 @@ func createParentOpt(clicontext *cli.Context, pipeFDEnvKey, stateDirEnvKey, pare
 	ipv6 := clicontext.Bool("ipv6")
 	if ipv6 {
 		logrus.Warn("ipv6 is experimental")
-		if s := clicontext.String("net"); s != "slirp4netns" {
+		if s := clicontext.String("net"); s != "pasta" && s != "slirp4netns" {
 			logrus.Warnf("--ipv6 is discarded for --net=%s", s)
 		}
 	}
 
 	disableHostLoopback := clicontext.Bool("disable-host-loopback")
 	if !disableHostLoopback && clicontext.String("net") != "host" {
```

Comment:
I think there is a problem here , It should be `if disableHostLoopback && clicontext.String("net") != "host"` right ?